### PR TITLE
Bug 1476546 - Convert lodash .isString() to native ES6 JS

### PR DESCRIPTION
**Pull request by Soundharyaam**
lodash .isString() has been converted to ES6

### CODE REVIEW

**Comment by edmorley** [comment]
Thank you for the PR! We try to keep PRs to be just about one bug number - could you split out the `padStart` changes, and make this just about `isString` (bug 1476546)? No need to close this PR, just update the commit (I'd squash them locally too), and force push to the PR / update the PR title.

**Comment by Soundharyaam** [comment]
Sure. I will do that. Do I have to create different branch for each Bug or is it okay work on the same branch? If I have to work on same branch for all bugs, should I have to wait until each one of them is merged?

**Comment by edmorley** [comment]
A pull request has to come from a branch. So to have several pull requests, you need several branches. Having them open at the same time is fine (sometimes the changes in each will conflict, aka overlap, but they shouldn't in this case).

**Comment by Soundharyaam** [comment]
Thanks. I will make the required changes soon.

File: ui/js/controllers/perf/graphs.js [hunk 1]
```diff
@@ -762,7 +762,7 @@ perf.controller('GraphsCtrl', [
 
             if ($stateParams.series) {
                 $scope.seriesList = [];
-                if (_.isString($stateParams.series)) {
+                if (typeof ($stateParams.series) === 'string') {
```
Comment (edmorley) [inline ui/js/controllers/perf/graphs.js]: The parentheses around `$stateParams.series` are not needed, could you remove them?

**Pull request merged**